expression/builtin_string.go

```diff
@@ -3491,5 +3493,67 @@ type loadFileFunctionClass struct {
 }
 
 func (c *loadFileFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
-	return nil, errFunctionNotExists.GenWithStackByArgs("FUNCTION", "load_file")
+	if err := c.verifyArgs(args); err != nil {
+		return nil, err
+	}
+	valStr, _ := ctx.GetSessionVars().GetSystemVar(variable.MaxAllowedPacket)
+	maxAllowedPacket, err := strconv.ParseInt(valStr, 10, 64)
```

Comment:
According to the MySQL document [LOAD_FILE(file_name)](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_load-file), we should also check whether the user has the `FILE` privilege:

> you must have the FILE privilege.

Reply:
@zz-jason  MySQL has two checks "file_secure_path" and "FILE privilege" , both seems not exists in TiDB now, maybe it's better to release this func after more compicated privilege system

Reply:
> MySQL has two checks "file_secure_path" and "FILE privilege" , both seems not exists in TiDB now

@tiancaiamao Could you help @cfzjywxk to add the `FILE` privilege?

Reply:
priv system is another task, is there a detailed design or implementation plan or roadmap about privilege system in TiDB ?   or we just make it full compatible with MySQL privilege system?
@zz-jason @tiancaiamao

Reply:
Yeah, we can do it in another PR. Just to mention @tiancaiamao to notice that we haven't implemented the `FILE` privilege. 😄

Reply:
@cfzjywxk How about filing an issue about this improvement?